### DISCUSSION TITLE
Desktop Agent and stopping the flow

### DISCUSSION THREAD

**Discussion by BrianJonesNMI** [postmanlabs/postman-flows#40] [Q&A]
I am trying to use Merge which apparently is not available in the current desktop version. I have installed the desktop agent but the flow is just sitting on the external request and not moving forward. Nothing in the console either.

Is the Desktop agent supported in the flows?

After starting the flow the system stalls at the first external request. The animated gif oscillates and does not stop. How can we stop a flow when it is not performing as desired?

**Comment by anudeepmedicharla** [comment]
Hey, we might need more information to understand why the flow is not moving forward after one request. But for now, to stop a flow, you can right-click -> Refresh anywhere on the canvas. That should reset the flow.

**Reply by BrianJonesNMI** [reply]
I have engages support on the request issue, it seems to be related to the Desktop Agent.

Thanks for the refresh tip it does stop the flow.

Needing the merge changes in the desktop version as soon as you all can. Many of us use non-public APIs and the CORS issues with the browser tool set makes things difficult

**Reply by samay-v** [reply]
> Needing the merge changes in the desktop version as soon as you all can. Many of us use non-public APIs and the CORS issues with the browser tool set makes things difficult

Can you try using the Desktop Agent on the web app, that should solve your CORS issue. Learn more about agents [here](https://learning.postman.com/docs/getting-started/installation-and-updates/#installing-the-postman-desktop-agent).

Rest assured, we're working on making sure the desktop version stays updated with the web app

**Comment by BrianJonesNMI** [comment]
I am trying the agent and I have an open support ticket as the agent is not working.
